presto-main/src/main/java/com/facebook/presto/metadata/FunctionAndTypeManager.java

```diff
@@ -301,6 +325,17 @@ public FunctionMetadata getFunctionMetadata(FunctionHandle functionHandle)
         return functionNamespaceManager.get().getFunctionMetadata(functionHandle);
     }
 
+    @Override
+    public Type instantiateParametricType(ExactTypeSignature exactSignature)
```

Comment:
Can this take in a `TypeSignature`, and we manually create an `ExactTypeSignature` inside this method, so we can avoiding moving `ExactTypeSignature` into the SPI?

Reply:
That would mean not using the `ExactTypeSignature` here: https://github.com/prestodb/presto/blob/fa3a49b8d6df5625865506df8d176cb457de2796/presto-native-sidecar-plugin/src/main/java/com/facebook/presto/sidecar/typemanager/NativeTypeManager.java#L142,  which makes the usage of `ExactTypeSignature` redundant.

Reply:
Done.